content/docs/user-guide/index.md

```diff
@@ -1,51 +1,82 @@
-# Overview
+# What is DVC?
+
+**Data Version Control** is a [free], open-source tool for [data management], ML
+[workflow automation], and [experiment management]. DVC takes advantage of the
+existing software engineering toolset your team already knows (Git, your IDE,
+CI/CD, etc.). DVC helps data science and machine learning teams manage large
+datasets, make projects reproducible, and collaborate better.
+
+[data management]: /doc/start/data-management
```

Comment:
Minor, but just curious: why not link to the UG here?

Reply:
My bad. I think I just copy/pasted this from existing content. Should be to the subpage indeed:

```suggestion
[data management]: /doc/user-guide/data-management
```